### PR TITLE
Add organisation and reflection as appendix

### PR DESCRIPTION
**Pull request by jzzheng22**

### CODE REVIEW

**Review by daryllimyt — DISMISSED** [review]
LGTM

**Review by SaraFFdez — DISMISSED** [review]
LGTM

**Review by daryllimyt — DISMISSED** [review]

File: 015_appendix/015_appendix.tex [hunk 1]
```diff
@@ -1,3 +1,128 @@
...
+    \item Christoph Renschler
+    \item Adrian Koch
+    \item Vasileios Manginas
+\end{itemize}
+
+Team 7: Agent Design
+\begin{itemize}
+    \item \textbf{Moin Bukhari}
+    \item Nayan Kad
+    \item Abdullah ehsan
```
Comment (SaraFFdez) [inline 015_appendix/015_appendix.tex]:
```suggestion
    \item Abdullah Ehsan
```

**Review by SaraFFdez — COMMENTED** [review]
just a small change

**Review by SaraFFdez — APPROVED** [review]
LGTM

**Review by daryllimyt — APPROVED** [review]

**Pull request merged**